<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>84</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="continueButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>250</y>
     <width>84</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Continue</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="minutesSpinBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>43</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="secondsSpinBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>190</y>
     <width>43</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="minutesLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>63</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>minutes</string>
   </property>
  </widget>
  <widget class="QLabel" name="secondsLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>189</y>
     <width>63</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>seconds</string>
   </property>
  </widget>
  <widget class="QLabel" name="gameLengthLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Game length</string>
   </property>
  </widget>
  <widget class="QLabel" name="welcomeLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Welcome! Please select game settings below.</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>118</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="realTimeRadioButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>171</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Use real time (default)</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="manualTimeRadioButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>141</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Set time manually</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="playerNameEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>8</number>
   </property>
  </widget>
  <widget class="QLabel" name="playerNameLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Player name:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>continueButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
